tests/UntrustedPSPIntegrationTests.js

```diff
@@ -206,102 +206,100 @@ gpii.tests.pspIntegration.saveTestDefs = [
-                    listener: "fluid.identity"
-                }, {
-                    func: "{pspClient}.connect"
-                }, {
-                    event: "{pspClient}.events.onConnect",
-                    listener: "gpii.tests.pspIntegration.connectionSucceeded"
-                }, {
-                    event: "{pspClient}.events.onReceiveMessage",
-                    listener: "gpii.tests.pspIntegration.checkPayload",
-                    args: ["{arguments}.0", "modelChanged"]
-                }, {
-                    funcName: "gpii.tests.pspIntegration.sendMsg",
-                    args: [ "{pspClient}", [ "preferences","http://registry\\.gpii\\.net/common/pitch"], 0.85]
-                }, {
-                    event: "{pspClient}.events.onReceiveMessage",
-                    listener: "gpii.tests.pspIntegration.checkPayload",
-                    args: ["{arguments}.0", "modelChanged"]
-                }, {
-                    event: "{pspClient}.events.onReceiveMessage",
-                    listener: "gpii.tests.pspIntegration.checkPayload",
-                    args: ["{arguments}.0", "preferencesApplied"]
-                }, {
-                    func: "gpii.test.checkConfiguration",
-                    args: ["{tests}.options.data.initial.settingsHandlers", "{nameResolver}", "{testCaseHolder}.events.onCheckConfigurationComplete.fire"]
-                }, {
-                    event: "{testCaseHolder}.events.onCheckConfigurationComplete",
-                    listener: "fluid.identity"
-                }, {
-                    func: "{rawPrefsAfterAutoSave}.send"
-                }, {
-                    event: "{rawPrefsAfterAutoSave}.events.onComplete",
-                    listener: "gpii.test.untrusted.pspIntegration.verifyRawPrefsAfterAutoSave",
-                    args: ["{that}", "{arguments}.0"]
-                }, {
-                    funcName: "gpii.tests.pspIntegration.sendMsg",
-                    args: [ "{pspClient}", [ "saveButtonClickCount" ], 1]
-                }, {
-                    event: "{pspClient}.events.onReceiveMessage",
-                    listener: "gpii.tests.pspIntegration.checkPayload",
-                    args: ["{arguments}.0", "preferencesApplied"]
-                }, {
-                    func: "{rawPrefsAtEnd}.send"
-                }, {
-                    event: "{rawPrefsAtEnd}.events.onComplete",
-                    listener: "gpii.test.untrusted.pspIntegration.verifyRawPrefsAtEnd",
-                    args: ["{that}", "{arguments}.0", null, {
-                        "flat": {
-                            "contexts": {
-                                "gpii-default": {
-                                    "name": "Default preferences",
-                                    "preferences": {
-                                        "http://registry.gpii.net/common/pitch": 0.85
-                                    }
+            {
```

Comment:
See above

Reply:
We should chat about it, it feels appropriate to me to change the indentation for multi-element arrays, especially if you're reorganising them anyway.